doc/Guide to Contributing to Sia.md

```diff
@@ -0,0 +1,175 @@
+# Contributing to Sia
+
+#### Table of Contents
+* [Get started with Go](#go)
+  * [Install Go](#install-go)
+  * [Learn Go]("learn-go")
+* [Build Sia](#build-sia)
+* [Contribute to the codebase](#contribute)
+  * [Set up git](#setup-git)
+  * [Fork the Sia repository](#fork)
+  * [Write some code](#write)
+  * [Make a pull request](#pull)
+  * [More git resources](#git-resources)
+* [Contact us](#contact)
+
+<a name="go"/>
+## Get started with Go
+
+<a name="install-go"/>
+### Install Go
+To install Go on your computer, follow the [official installation guide][install-go].  
+
+You should install the latest [official Go binary][binary] for your system (if not available, [install from source][source]).
+If you plan to cross compile Sia, see [Cross Compilation with Go 1.5][cross] by Dave Cheney.  
+
+Now make a workspace directory in which you will store source code and dependencies.  You can choose any filepath except where you installed Go (don't choose `/usr/local`).
+```bash
+# make a working directory called golang in your home directory
+$ mkdir $HOME/golang
+# store base path in an environmental variable
+$ echo 'export GOROOT=$HOME/golang' >> $HOME/.profile
```

Comment:
shouldn't this be `$GOPATH`? `$GOROOT` is where the Go tools and stdlib are stored.